lidiabrowser/sync/populate.py

```diff
@@ -17,11 +20,49 @@
     Publication,
     TermGroup,
 )
+from sync.zoteroutils import get_attachment_url, get_attachment_id_from_url
+
 
 logger = logging.getLogger(__name__)
 
 
 LIDIAPREFIX = "~~~~LIDIA~~~~"
+LEXICON_URLS = {}
+
+
+def fetch_lexicon_data():
+    url = "https://github.com/CentreForDigitalHumanities/lidia-zotero/raw/main/vocabulary/lexicon.xlsx"
+    filename = "../data/lexicon.xlsx"
+
+    if os.path.isfile(filename) and os.path.getsize(filename) > 0:
+        logger.info("Lexicon spreadsheet already downloaded.")
+        return
+
+    os.makedirs(os.path.dirname(filename), exist_ok=True)
+
+    try:
+        with urllib.request.urlopen(url) as response:
+            with open(filename, 'wb') as out_file:
+                shutil.copyfileobj(response, out_file)
+        logger.info("Lexicon spreadsheet downloaded successfully.")
+    except Exception as e:
+        logger.error(f"Error downloading lexicon spreadsheet: {e}")
+
+
+def load_lexicon_data():
+    workbook = openpyxl.load_workbook('../data/lexicon.xlsx')
+    sheet = workbook.active
```

Comment:
Make sure to select the correct worksheet, because there are two and the active one is the one that the user last selected...: `workbook['entries']`

Reply:
Good point